crates/common/consensus/src/execution_engine/mod.rs

```diff
@@ -166,4 +169,26 @@ impl ExecutionEngine {
             .await?
             .to_result()
     }
+
+    pub async fn engine_forkchoice_updatedv3(
```

Comment:
```suggestion
    pub async fn engine_forkchoice_updated_v3(
```

Could you update the other functions so they have a _v3 instead of just v3